nodeJS/express/forms_and_data_handling.md

```diff
@@ -541,11 +541,10 @@ Don't forget to update the view to display these new fields!
 
 What if we want to search for a specific user in a list of thousands? We'll need a new route and view that lets clients search our list of users.
 
-1. Add a form (in `createUser.ejs` or another view) which accepts a `name` or `email` (or both!)
-1. Create a new route `/search` which accepts `GET` and `POST` requests.
-1. Add the search logic to your controller which searches your list for a matching user.
-   - Your `POST` request should handle searching for the user.
-   - Your `GET` request should then render the search result.
+1. Add a form with a `GET` method (in `createUser.ejs` or another view) which accepts a `name` or `email` (or both!)
+1. Create a new route `/search` which accepts a `GET` request.
+1. Add the search logic to your controller which searches your list for a matching user. Form data that has been sent via a `GET` request will not be available in the `req.body`. You will need to use `req.query` instead.
```

Comment:
Nit
```suggestion
1. Add the search logic to your controller which searches your list for a matching user. Form data that has been sent via a `GET` request will not be available via `req.body`. You will need to use `req.query` instead.
```

Reply:
Done! The extra space is gone and `in the` has been changed to `via`.